### PR TITLE
Fix multiple locations/times

### PR DESCRIPTION
**Pull request by sidpagariya**
...if there are more than two lines for a single course in the Schedule section.

### CODE REVIEW

File: Schedule-maker/schedule-parser.js [hunk 1]
```diff
@@ -47,21 +47,27 @@ function parseSchedule(){
...
-        try {
-            time.from = convertTime(sched[0].split(" ")[1]);
-            time.to = convertTime(sched[0].split(" ")[3]);
-        }
-        catch(err) {
-            // this means time is not provided
-            continue;
+        subscheds = [];
+        locs = [];
+        for (var j=0; j<sched.length/2; j++){
```
Comment (j-usti-n) [inline Schedule-maker/schedule-parser.js]: Not sure I'f I'm completely understanding, but shouldn't this be `j += 2` instead of `j++`?

Comment (sidpagariya) [reply Schedule-maker/schedule-parser.js]: OH yooo you are rightttt!!

File: Schedule-maker/schedule-parser.js [hunk 2]
```diff
@@ -49,7 +49,7 @@ function parseSchedule(){
         var sched = tbodymain.children[i].children[1+offset].firstElementChild.innerText.split('\n');
         subscheds = [];
         locs = [];
-        for (var j=0; j<sched.length/2; j++){
+        for (var j=0; j<sched.length; j+=2){
```
Comment (sidpagariya) [inline Schedule-maker/schedule-parser.js]: Now how about this @kulaj?

**Review by sidpagariya — COMMENTED** [review]
It should be good now @kulaj what do you think, and how about you @NoahFiner?

**Pull request merged**